src/components/screen/ChangePw.tsx

```diff
@@ -38,26 +40,44 @@ function ChangePw(props: Props): ReactElement {
   const [currentPw, setCurrentPw] = useState('');
   const [newPw, setNewPw] = useState('');
   const [confirmPw, setConfirmPw] = useState('');
-  const close = (): void => {
-    navigation.goBack();
-  };
+  const [mutationChangePassword] = useMutation<{ changeEmailPassword: boolean }, MutationChangePasswordInput>(
```

Comment:
`mutationChangePassword` ==> `changePassword` would be better naming

Reply:
I changed the name of the variables. Thanks!